packages/bitcore-wallet-service/README.md

```diff
@@ -70,211 +70,329 @@ BWS can be used with PM2 with the provided `app.js` script:
 ```
 
 @dabura667 made a report about how to use letsencrypt with BWS: https://github.com/bitpay/bitcore-wallet-service/issues/423
-  
+
+# Enabling Regtest Mode
```

Comment:
Can you reset --hard from master and only pull in this change? There are quite a few files changed in this MR, and I feel like this README change is really the only thing needed